engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/EngineJobHashingActor.scala

```diff
@@ -104,8 +104,10 @@ case class EngineJobHashingActor(receiver: ActorRef,
       HashResult(HashKey(s"output expression: ${output.wdlType.toWdlString} ${output.name}"), output.requiredExpression.valueString.md5HashValue)
     }
 
-    // Build these all together for the final set of initial hashes:
-    Set(commandTemplateHash, backendNameHash, inputCountHash) ++ runtimeAttributeHashes ++ inputHashResults ++ outputExpressionHashResults
+    val outputCountHash = HashResult(HashKey("output count"), jobDescriptor.call.task.outputs.size.toString.md5HashValue)
```

Comment:
Actually, I have this in my PR, and I had to fix a bunch of EJHA tests (which I don't see here...). Feel free to undo this change and let me cover it when I push my PR